### PR TITLE
More public XmlPath members

### PR DESCRIPTION
**Pull request by rmboggs**
This makes the constructors and a couple of static methods in the XmlPath class public.

### CODE REVIEW

**Comment by twsouthwick** [comment]
Do we need both the constructor and the factory?

**Comment by rmboggs** [comment]
> Do we need both the constructor and the factory?

Honestly, I think the constructors would be sufficient.  I only did that because @ThomasBarnekow suggested it.  If you need me to revert the factory, let me know.

**Comment by twsouthwick** [comment]
Let's just do the constructors

**Comment by rmboggs** [comment]
> Let's just do the constructors

Fair enough, let me revert the non-constructor related changes

**Comment by rmboggs** [comment]
That should do it.

File: src/DocumentFormat.OpenXml/XmlPath.cs [hunk 1]
```diff
@@ -21,9 +21,13 @@ namespace DocumentFormat.OpenXml
     public class XmlPath
     {
         /// <summary>
-        /// Initializes a new instance of the XmlPath.
+        /// Initializes a new instance of the XmlPath from the specified
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml/XmlPath.cs:24]: can you use `<see href=""/>` annotations for types here?

File: src/DocumentFormat.OpenXml/XmlPath.cs [hunk 2]
```diff
@@ -54,7 +58,7 @@ internal XmlPath(OpenXmlElement element)
         /// Initializes a new instance of the XmlPath from the specified OpenXmlPart.
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml/XmlPath.cs:58]: same as above

File: src/DocumentFormat.OpenXml/XmlPath.cs (hunk 1 above)
Comment (rmboggs) [reply src/DocumentFormat.OpenXml/XmlPath.cs:24]: You want me to use `<see href=""/>` here or `<see cref=""/>`?  If the former, what url should I be using for the classes?

Comment (twsouthwick) [reply src/DocumentFormat.OpenXml/XmlPath.cs:24]: haha yeah cref

Comment (rmboggs) [reply src/DocumentFormat.OpenXml/XmlPath.cs:24]: Cool, just making sure.  I'll work on that.

Comment (rmboggs) [reply src/DocumentFormat.OpenXml/XmlPath.cs]: Should be updated as requested now.

**Review by twsouthwick — APPROVED** [review]

**Comment by rmboggs** [comment]
Hi,

Since this is approved, do you know when it can be merged into Main?

**Comment by twsouthwick** [comment]
Done

**Pull request merged**